lib/membrane_ffmpeg_swresample_plugin/converter.ex

```diff
@@ -128,7 +128,7 @@ defmodule Membrane.FFmpeg.SWResample.Converter do
     output_frame_size = RawAudio.frame_size(state.output_stream_format)
     ratio = output_frame_size / input_frame_size
 
-    expected_output_frames_count = byte_size(payload) / input_frame_size * ratio
+    expected_output_frames_count = (byte_size(payload) / input_frame_size * ratio) |> trunc()
```

Comment:
I think it's necessary to use `round`. Otherwise we'd expect 2137 frames if the float result is 2137.99